machine_usage/forms.py

```diff
@@ -78,15 +78,25 @@ def __init__(self, machine, *args, **kwargs):
 
 #Usage Length usage
 class MachineUsageLength(forms.Form):
-    usage_hours = forms.IntegerField(required=True, min_value=0, initial=0, widget=forms.NumberInput(attrs={'placeholder': 'Hours'}))
-    usage_minutes = forms.IntegerField(required=True, min_value=0, max_value=60, initial=0, widget=forms.NumberInput(attrs={'placeholder': 'Minutes'}))
+    usage_hours = forms.IntegerField(required=False, min_value=0, widget=forms.NumberInput(attrs={'placeholder': 'Hours'}))
+    usage_minutes = forms.IntegerField(required=False, min_value=0, max_value=60, widget=forms.NumberInput(attrs={'placeholder': 'Minutes'}))
 
     def clean(self):
         cleaned_data = super().clean()
         usage_hours = self.cleaned_data['usage_hours']
         usage_minutes = self.cleaned_data['usage_minutes']
-        if usage_minutes==0 and usage_hours==0:
+        
+        if not usage_minutes and not usage_hours:
+            raise ValidationError("Time cannot be empty")

```

Comment:
Does this return the error to the user? Or does it cause a 500 error?

Reply:
from testing, the user just can't continue to the next step